src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailedPackageMetadata.cs

```diff
@@ -16,8 +18,9 @@ public DetailedPackageMetadata()
         {
         }
 
-        public DetailedPackageMetadata(PackageSearchMetadataContextInfo serverData, PackageDeprecationMetadataContextInfo deprecationMetadata, long? downloadCount)
+        public DetailedPackageMetadata(PackageSearchMetadataContextInfo serverData, PackageDeprecationMetadataContextInfo deprecationMetadata, PackageItemViewModel packageItemViewModel, long? downloadCount)
```

Comment:
I am not an expert in WPF, but my online search suggests that it is not common for model to have a dependency on view model, and it should the other way around.

Reply:
Correct! modified